### PR TITLE
Fixed error in loss function

### PR DESCRIPTION
**Pull request by yashyenugu**
changed `cross_entropy = tf.keras.losses.BinaryCrossentropy(from_logits=True)` to `cross_entropy = tf.keras.losses.BinaryCrossentropy()` since the discriminator already has a sigmoid activation function

### CODE REVIEW

**Comment by jeffheaton** [comment]
Thanks! Good catch
